esmvaltool/config-references.yml

```diff
@@ -615,7 +615,6 @@ references:
   aura-tes: "Beer, R., IEEE Trans. Geosci. Rem. Sens., doi:10.1109/TGRS.2005.863716, 2006."
   ceres-syn1deg: "Wielicki et al., Bull. Amer. Meteor. Soc., doi: 10.1175/1520-0477(1996)077<0853:CATERE>2.0.CO;2, 1996."
   cru: "Harris, I. et al., Int. J. Climatol., 34, doi: 10.1002/joc.3711, 2014."
-  eppley-vgpm-modis: "Behrenfeld, M. J., & Falkowski, P. G. (1997). Photosynthetic rates derived from satellite-based chlorophyll concentration. Limnology and oceanography, 42(1), 1-20."
```

Comment:
any reason this line is removed?

Reply:
I don't recall removing it. I only added one line for the new data set.

Reply:
@valeriupredoi - I did a fresh checkout  on the config-references.yml file and a pull on version2_development and do not see the entry for eppley-vgpm. What could be wrong?